### ISSUE TITLE
More Themes Needed!

### ISSUE THREAD

**Issue by Adammatthiesen** [MatthiesenXYZ/astro-ghostcms#18]
**Is your feature request related to a problem? Please describe.**
We need more themes...

**Describe the solution you'd like**
I would like myself, and hopefully any willing members of the community to create new themes for this integration.  following the theme-default package as the example of how to export

**Comment by Adammatthiesen** [comment]
New Theme Added https://catppuccindark-demo.astro-ghostcms.xyz/

**Comment by Adammatthiesen** [comment]
New Theme Added https://brutal-demo.astro-ghostcms.xyz

**Comment by Adammatthiesen** [comment]
As I am prepping for a new Theme system I am going to close this issue.  A new issue will be brought up once the new system is in place
